swap_meet/vendor.py

```diff
@@ -1,2 +1,71 @@
+from .item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        inventory = [] if inventory is None else inventory
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        else:
+            self.inventory.remove(item)
+            return item 
```

Comment:
The else here is unnecessary and should be removed to reduce conditional complexity.

if the condition on line 13 is never True, then any remaining lines of code in the function will run. We don't need else to ensure the remove logic will execute. 

This also unindents the 'main' logic on lines 16-17 which emphasizes to other devs that this is the primary purpose of the function.
```suggestion
        self.inventory.remove(item)
        return item 
```